src/components/organisms/ActionsPane/ActionsPane.styled.tsx

```diff
@@ -29,8 +29,6 @@ export const ActionsPaneFooterContainer = styled.div`
 `;
 
 export const ActionsPaneMainContainer = styled.div<{$height: number}>`
-  height: ${({$height}) => $height}px;
```

Comment:
I think that removing this height won't apply the vertical scrolling anymore where needed...